testem.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed&nolint',
```

Comment:
I added the `nolint` query parameter in case we may install lint-related addons like `ember-cli-dependency-lint`, which does run in testing by default.